client/coral-sign-in/components/FakeComment.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
Yes. I though about that but if we are using Comment as a fake comment here we need to refactor all the other components as it is impossible to disable the actions that way.